### ISSUE TITLE
Implement an interpreter

### ISSUE THREAD

**Issue by vshymanskyy** [rianhunter/wasmjit#22]
Just creating this ticket as I'm interested in this functionality, and it's listed in the top of TODO list.

Thanks!

**Comment by rianhunter** [comment]
Cool, I see you might be interested in ESP/ARM hardware. An interpreter is not too hard to build but I have a few higher priority things to get to this spring. I could guide you through the process if this is a very high priority thing for you. I would also welcome an ARM or an ESP JIT :)

**Comment by vshymanskyy** [comment]
@rianhunter you're right, it is quite interesting to run it on different kinds of devices.
Interpreter would enable them all, while jit version requires more work.
I think we could bring more attention to the project by having at least the interpreter part working...

**Comment by rianhunter** [comment]
@vshymanskyy Yes, agree interpreter would be good but right now it doesn't serve my goals. Bringing attention to the project isn't a goal of the project

**Comment by vshymanskyy** [comment]
@rianhunter could ypo please update the readme to reflect your thoughts?
![image](https://user-images.githubusercontent.com/1506708/54750252-b51ede00-4bdf-11e9-8586-bcea2312ac47.png)

**Comment by rianhunter** [comment]
The README isn't incorrect.
